pilot/pkg/serviceregistry/kube/controller/ambientindex_test.go

```diff
@@ -168,50 +184,52 @@ func TestAmbientIndex(t *testing.T) {
 	fx.Clear()
 
 	createService(controller, "svc1", "ns1",
+		map[string]string{},
 		map[string]string{},
 		[]int32{80}, map[string]string{"app": "a"}, t)
 	// Service shouldn't change workload list
 	assertWorkloads("", "name1", "name2", "name3")
```

Comment:
It seems weird Lookup all returns 1,2,3 but "10.0.0.1" returns 1,2,svc1. All should be a superset I would think

Reply:
actually its in `""` on L200 but not here...?